build/build-netlify.sh

```diff
@@ -51,10 +51,10 @@ if [[ "$INCOMING_HOOK_TITLE" = "nightly" ]]; then
 fi;
 
 # Run Algolia if building master
-# if [[ "$CONTEXT" = "production" ]]; then
-# 	echo "Building Algolia index..."
-# 	ALGOLIA_API_KEY=$PROD_ALGOLIA_API_KEY bundle exec jekyll algolia --config _config_base.yml,_config_url.yml --builds-config _config_cockroachdb.yml
-# fi;
+if [[ "$CONTEXT" = "production" ]]; then
+	echo "Building Algolia index..."
+	ALGOLIA_API_KEY=$PROD_ALGOLIA_API_KEY bundle exec jekyll algolia --config _config_base.yml,_config_url.yml --builds-config _config_cockroachdb.yml
```

Comment:
```suggestion
	ALGOLIA_API_KEY=${PROD_ALGOLIA_API_KEY} bundle exec jekyll algolia --config _config_base.yml,_config_url.yml --builds-config _config_cockroachdb.yml"
```
This syntax is somewhat more thread-safe. Careful to be sure that $PROD_ALGOLIA_API_KEY is actually set unless it being unset wouldn't cause a problem. As it is, if it was empty it would just be the `bundle exec....` bit that runs and might be difficult to debug. Probably not a significant issue, but this code is not resilient against shell-scripting injection such as adding a `sudo` or `rm -rf /;` to it.